tests/features/search_and_filters/search_and_filters.spec.ts

```diff
@@ -0,0 +1,163 @@
+        await pages.overview.createOwner(
+          firstOwner,
+          noResultsOwnerText,
+          titleAdmin,
+          noResultsTitleText,
+        );
+        await pages.topPanel.clickTab('Catalog');
+        await pages.catalog.searchByTextInFilter('Owner', 'firstOwner');
+        expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/26
+     */
+    test('should display the expected item with Tag filter', async ({ steps: { pages } }) => {
+      await test.step('apply filter Tag', async () => {
+        await pages.catalog.searchByText(bookETLDataEntity);
+        await pages.catalog.clickOnListItem(bookETLDataEntity);
+        await pages.overview.createTag(tagFilterOption, noResultsTagText);
+        await pages.topPanel.clickTab('Catalog');
+        await pages.catalog.searchByTextInFilter('Tag', 'aqa_tag');
+        expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/70
+     */
+    test('should display the expected item with Type filter', async ({ steps: { pages } }) => {
+      await test.step('I go to the Transformers tab', async () => {
+        await pages.catalog.clickTab(transformersTag);
+      });
+      await test.step('apply filter Type', async () => {
+        await pages.catalog.openFilterWithInput(typeFilter);
+        await pages.catalog.searchByTextInFilter('Type', 'JOB');
+        expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/71
+     */
+    test('should display the expected item with Group filter', async ({ steps: { pages } }) => {
+      await test.step('apply filter Group', async () => {
+        await pages.catalog.openFilterWithInput(groupsFilter);
+        await pages.catalog.chooseOption(groupsOption);
+        expect(await pages.catalog.isListItemVisible(postcardsAQADataEntity)).toBeTruthy();
+        expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+      });
+    });
+    test.describe('when apply multiply filter', () => {
+      test.beforeEach(async ({ steps: { pages }, page }) => {
+        await test.step(`I open catalog page`, async () => {
+          await page.goto('');
+          await pages.topPanel.clickTab('Catalog');
+        });
+      });
+      /**
+       * /project/1/test-cases/28
+       */
+      test('should display the expected item with different types of filters', async ({
+        steps: { pages },
+      }) => {
+        await test.step('apply filter Tag', async () => {
+          await pages.catalog.openFilterWithSelect(namespaceFilter);
+          await pages.catalog.chooseOption(namespaceOption);
+          expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+          expect(await pages.catalog.isListItemVisible(bookETLDataAQAEntity)).toBeTruthy();
```

Comment:
UPD: I think we can get rid of all assert except 'BookETL'.

This will make tests less dependent on test data

Reply:
fixed